composer.json

```diff
@@ -1,9 +1,16 @@
 {
-  "name": "nhsleadership/nhsblocks",
-  "description": "Plugin for WordPress to add NHSUK Frontend style components to Gutenberg",
-  "repositories":[{"type":"composer","url":"https://wpackagist.org"}],
-  "type": "wordpress-plugin",
-  "require": {
-    "composer/installers": "^1.3"
-  }
+    "name": "nhsleadership/nhsblocks",
+    "description": "Plugin for WordPress to add NHSUK Frontend style components to Gutenberg",
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://wpackagist.org"
+        }
+    ],
+    "type": "wordpress-plugin",
+    "minimum-stability": "stable",
```

Comment:
Please remove "minimum-stability": - we do nt need it as it is a meta package (no requirements)